.buildkite/scripts/steps/code_coverage/jest_parallel.sh

```diff
@@ -13,9 +13,15 @@ exitCode=0
 while read -r config; do
   if [ "$((i % JOB_COUNT))" -eq "$JOB" ]; then
     echo "--- $ node scripts/jest --config $config --coverage --coverageReporters json --coverageDirectory target/kibana-coverage/jest"
-    node --max-old-space-size=14336 ./node_modules/.bin/jest --runInBand --config="$config" --coverage --coverageReporters json --coverageDirectory target/kibana-coverage/jest || true
-    echo "Rename coverage-final.json to avoid overwrite"
-    mv target/kibana-coverage/jest/coverage-final.json "./target/kibana-coverage/jest/coverage-$(date +%s%3N).json"
+    node --max-old-space-size=14336 ./node_modules/.bin/jest --runInBand --config="$config" \
+      --coverage --coverageReporters json --coverageDirectory target/kibana-coverage/jest \
+      --passWithNoTests || true
+    if [[ -f "target/kibana-coverage/jest/coverage-final.json" ]]; then
+      echo "Rename coverage-final.json to avoid overwrite"
+      mv target/kibana-coverage/jest/coverage-final.json "./target/kibana-coverage/jest/coverage-$(date +%s%3N).json"
```

Comment:
@wayneseymour it is generating a timestamp like `1647448677547`. Since we run multiple jest configs, code coverage json for each of them has the same `coverage-final.json` filename. To avoid file override (jest configs are run sequentially) and actually collect all of them for later merging I'm doing renaming and timestamp is reasonable solution.